control-plane/agents/src/bin/core/tests/event/mod.rs

```diff
@@ -0,0 +1,217 @@
+use deployer_cluster::{Cluster, ClusterBuilder};
+use events_api::{
+    event::{EventAction, EventCategory, EventMessage},
+    mbus_nats::{message_bus_init, BusSubscription},
+    Bus,
+};
+use grpc::operations::{
+    nexus::traits::NexusOperations, pool::traits::PoolOperations, volume::traits::VolumeOperations,
+};
+use std::{collections::HashMap, time::Duration};
+use stor_port::types::v0::{
+    openapi::{
+        models,
+        models::{CreateVolumeBody, PublishVolumeBody, VolumePolicy},
+    },
+    transport::{DestroyNexus, DestroyPool, DestroyVolume, NexusId, UnpublishVolume, VolumeId},
+};
+use tokio::time::timeout;
+
+const CHILD_WAIT: u64 = 5;
+
+async fn build_cluster(num_ioe: u32, pool_size: u64) -> Cluster {
+    let reconcile_period = Duration::from_millis(300);
+    let child_twait = Duration::from_secs(CHILD_WAIT);
```

Comment:
Are we waiting for a full rebuild here? If so let's make this a **lot** smaller than 5s...

Reply:
Yes. Changed it to 1 milli sec.